app/controllers/application_controller.rb

```diff
@@ -77,6 +78,11 @@ def current_session
     @current_session ||= SessionActivation.find_by(session_id: cookies.signed['_session_id'])
   end
 
+  def current_theme
+    return Setting.default_settings['theme'] unless Themes.instance.names.include? current_user&.setting_theme
```

Comment:
I think `return current_user.setting_theme if Themes.instance.names.include? current_user&.setting_theme` is easier to understand.